### PR TITLE
Task #0000 feat: Update the event List and filter view

### PR DESCRIPTION
**Pull request by komalm**

### CODE REVIEW

File: src/app/demo/demo.component.ts [hunk 1]
```diff
@@ -1,174 +1,310 @@
...
+  isLoading: boolean =  true;
+  myEvents: any[];
+  myEventsCount : any;
+  p: number = 1;
+  collection: any[];
+  //event-calender parameter
   eventCalender: any;
-  events: MyCalendarEvent[];
+  events: CalendarEvent[];
+  eventIdentifier = "do_2133909787676262401550";
```
Comment (paritshivani) [inline src/app/demo/demo.component.ts:62]: Need to remove the harcoded values

File: src/app/demo/demo.component.ts [hunk 2]
```diff
@@ -1,174 +1,310 @@
...
   tommorrowDate = this.today.getFullYear() + '-' + ('0' + (this.today.getMonth() + 1)).slice(-2) + '-' + ('0' + (this.today.getDate()+1)).slice(-2);
-
-  p: number = 1;
-  eventIdentifier = 'do_11322166143296307218';
-  collection: any[];
+  tempFilterKeyName :any;
+  sort_by:any;
+  todaysCalenderEvent: any[];
+  todaysDate: any;
+  tempFlag?: any;
```
Comment (paritshivani) [inline src/app/demo/demo.component.ts:79]: variable name does not clear motive

File: src/app/demo/demo.component.ts [hunk 3]
```diff
@@ -1,174 +1,310 @@
...
-    private sbToastService: SbToastService)
-    {}
+    private sbToastService: SbToastService,
+    private libEventService: LibEventService )
+  { }
 
   ngOnInit() {
+    this.eventConfig = _.get(this.libEventService.eventConfig, 'context.user');
+    this.userId = this.eventConfig.id;
+    this.eventDetailService.getEvent('do_213441736769036288135').subscribe((data: any) => {
```
Comment (paritshivani) [inline src/app/demo/demo.component.ts:93]: Need to remove hardcoded id

**Comment by paritshivani** [comment]
@AnkitaMaheshChavan , As per our discussion on call please resolve the comments on this PR in your next PR.

File: src/app/demo/demo.component.ts (hunk 3 above)
Comment (AnkitaMaheshChavan) [reply src/app/demo/demo.component.ts:93]: removed this code from file

File: src/app/demo/demo.component.ts (hunk 1 above)
Comment (AnkitaMaheshChavan) [reply src/app/demo/demo.component.ts:62]: removed

File: src/app/demo/demo.component.ts (hunk 2 above)
Comment (AnkitaMaheshChavan) [reply src/app/demo/demo.component.ts:79]: tempFlag is chanaged to tempFlagCalendarFilter

**Pull request merged**